cognee/api/v1/responses/dispatch_function.py

```
@@ -0,0 +1,107 @@
+import json
+import logging
+from typing import Any, Dict, Union
+
+from cognee.api.v1.responses.models import ToolCall
+from cognee.modules.search.types import SearchType
```

Ruff (0.8.2)

`cognee.modules.search.types.SearchType` imported but unused

Remove unused import: `cognee.modules.search.types.SearchType`

(F401)